Aquality.Selenium.Core/tests/Aquality.Selenium.Core.Tests/Utilities/JsonFileTests.cs

```diff
@@ -1,42 +1,107 @@
 ﻿using Aquality.Selenium.Core.Utilities;
 using NUnit.Framework;
 using System;
+using System.Collections.Generic;
+using Aquality.Selenium.Core.Configurations;
 
 namespace Aquality.Selenium.Core.Tests.Utilities
 {
-    [NonParallelizableAttribute]
+    [NonParallelizable]
```

Comment:
seems that some of them could be parallelizable now :)

Reply:
updated